cypress/integration/advanced-functionalities/pagination/compose/Record_list.spec.js

```diff
@@ -0,0 +1,97 @@
+// Before running this test make sure to first run these following tests so you'll have resources to work with:
+// 1) cypress/integration/basic-functionalities/compose/namespace/Create.spec.js
+// 2) cypress/integration/basic-functionalities/compose/module/Create.spec.js
+// 3) cypress/integration/basic-functionalities/compose/record-page/Create.spec.js
+// 4) cypress/integration/basic-functionalities/compose/record-page/Edit.spec.js
+// 5) cypress/integration/basic-functionalities/compose/record/Create.spec.js
+
+/// <reference types="cypress" />
+const composeURL = Cypress.env('COMPOSE_URL')
+const email = Cypress.env('USER_EMAIL')
+const password = Cypress.env('USER_PASSWORD')
+
+describe('Testing pagination of compose record list', () => {
+  before(() => {
+    if (!window.sessionStorage.getItem('auth.refresh-token')) {
+      cy.login({ email, password, url: composeURL })
+    }
+  })
+
+  context('Testing pagination of compose record list', () => {
+    it('should be able to show total record count and limit the records per page', () => {
+      cy.visit(composeURL + '/namespaces')
+      cy.get('[data-test-id="input-search"]').type('cypress')
+      cy.get('[data-test-id="link-visit-namespace-cypress_namespace"]').click({ force: true })
+      // We wait 2s in order the page to be fully loaded
+      cy.wait(2000)
+      cy.get('[data-test-id="button-page-builder"]').click()
+      cy.get('[data-test-id="button-edit"]').click()
+      //cy.get('[data-test-id="tab-record-list"]').click({ force: true })
+      cy.get('.modal-content > .modal-body > .tabs > div > .nav-tabs > li:eq(1)').click()
+      cy.get('[data-test-id="link-select-all"]').click()
+      cy.get('[data-test-id="input-records-per-page"]').clear().type('2')
+      cy.get('[data-test-id="show-total-record-count"]').check({ force: true })
+      // We click on Save and close
+      cy.get('.modal-footer > .btn-primary').click()
+      cy.get('[data-test-id="button-save-and-close"]').click()
+      // We wait 2s in order the page to be fully loaded
+      cy.wait(2000)
+      cy.get('[data-test-id="button-public"]').click()
+      // We wait 1s in order the page to be fully loaded
+      cy.wait(1000)
+      cy.get('.card-footer').should('contain', '7 records')
+      cy.get('[data-test-id="next-page"]').click()
+      // When on second page we check if the record count is changed
+      cy.get('.card-footer').should('contain', '3 - 4')
+    })
+
+    it('should be able to display one record count if one record is present', () => {
+      cy.get('.card').within(() => {
```

Comment:
There's another (better) class to select by. Check all of the classes that this div has